unreleased/docker-multi-arch.yaml

```
@@ -0,0 +1,15 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: otelcontribcol
+
+# A brief description of the change
+note: Enabled multi-arch build for docker build
```

I think this might confuse readers. Can this state that this is for dev purposes?

```suggestion
note: Enabled multi-arch build for local development container builds
```

---

Done